client/package.json

```diff
@@ -4,7 +4,9 @@
   "description": "Experts portal client",
   "main": "index.js",
   "scripts": {
-    "start": "node serve.js"
+    "start": "node serve.js",
+    "dist": "rm -rf dist/ && webpack -p --config webpack.production.config.js",
```

Comment:
Don't you have `dist/redirect.html` there? You killing it every time on build then?